site/layouts/team/single.html

```diff
@@ -79,6 +79,18 @@ <h1 class="text--white">ZAP Team: {{ .Params.name }}</h1>
               </td>
             </tr>
             {{ end }}
+            {{ if .Params.bluesky }}
+            <tr>
+              <td>
+                <strong>Bluesky</strong>
+              </td>
+              <td>
+                <a rel="me" href="{{ .Params.bluesky }}">
```

Comment:
Is the me functional here too?

Reply:
Probably not. I'll remove that, I hadn't compared when I copied the mastodon block.

Thanks